paasta_tools/check_chronos_jobs.py

```diff
@@ -42,8 +52,14 @@ def compose_monitoring_overrides_for_service(chronos_job_config, soa_dir):
     monitoring_overrides['check_every'] = '1m'
     monitoring_overrides['runbook'] = monitoring_tools.get_runbook(
         monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
+
+    if 'realert_every' not in monitoring_overrides:
+        guessed_realert_every = _guess_realert_every(chronos_job_config)
+        if guessed_realert_every is not None:
```

Comment:
Ok we may have one last problem with this approach: how will users expect this to interact with "alert_after" ?

I think it will be ok, but could be surprising to people who have long jobs (daily) and an alert_after with the default 2m.

Because on the first failure detection, it will be suppressed because it hasn't been 2m. Which means they will have to wait another 24h before the alert goes off.

I think for chronos jobs the alert_after should default to 0, or something else, I'm not 100% sure on what to do.